rakefile.netstd.rb

```diff
@@ -48,15 +49,13 @@
 ]
 
 repo = 'FakeItEasy/FakeItEasy'
-teamcity_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 appveyor_server_url = 'https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings'

```

Comment:
Good eye, @thomaslevesque. Fixing sounds good, but we've published the old URL for the Appveyor feed. So…